src/modules/module_06211.c

```diff
@@ -236,7 +236,7 @@ int module_hash_decode (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSE
 
   // fake digest
 
-  memcpy (digest, tc->data_buf, TC_DATA_LEN / 4);
+  memcpy (digest, tc->data_buf, sizeof(tc->data_buf));
```

Comment:
Due to convention there should be space between the function name and the bracket.

```suggestion
  memcpy (digest, tc->data_buf, sizeof (tc->data_buf));
```